vita3k/kernel/src/sync_primitives.cpp

```diff
@@ -1358,6 +1368,25 @@ SceUID eventflag_create(KernelState &kernel, const char *export_name, SceUID thr
     return uid;
 }
 
+SceUID eventflag_find(KernelState &kernel, const char *export_name, const char *pName) {
+    if (strlen(pName) > KERNELOBJECT_MAX_NAME_LENGTH)
+        return RET_ERROR(SCE_KERNEL_ERROR_UID_NAME_TOO_LONG);
+
+    if (LOG_SYNC_PRIMITIVES)
+        LOG_DEBUG("{}: name: \"{}\"", export_name, pName);
+
+    const std::lock_guard<std::mutex> kernel_lock(kernel.mutex);
+
+    const auto it = std::find_if(kernel.eventflags.begin(), kernel.eventflags.end(), [=](const auto &eventflag) {
+        return strncmp(eventflag.second->name, pName, KERNELOBJECT_MAX_NAME_LENGTH) == 0;
+    });
+
+    if (it != kernel.eventflags.end())
+        return it->first;
+
+    return RET_ERROR(SCE_KERNEL_ERROR_UNKNOWN_EVF_ID);
```

Comment:
Are you sure this error is right ?

Reply:
Yes is same error other, it return unknow (type) id

Reply:
I'm quite sure it would return SCE_KERNEL_ERROR_UID_CANNOT_FIND_BY_NAME

Reply:
Hmm i have juste used error already pressent for o’e already here 🤔 but maybe yes

Reply:
i have done and changed for all